src/engagement_db_to_rapid_pro/engagement_db_to_rapid_pro.py

```diff
@@ -232,7 +232,7 @@ def sync_engagement_db_to_rapid_pro(engagement_db, rapid_pro, uuid_table, sync_c
 
     # Load all the project code schemes so we can easily scan for STOP messages later.
     code_schemes = []
-    for path in glob.glob("code_schemes/*.json"):
+    for path in glob.glob("**/*.json", recursive=True):
```

Comment:
Should this be code_schemes/**/*.json to make sure we're only (recursively) looking in the code_schemes directory?